cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/Host.java

```diff
@@ -182,14 +182,23 @@ Replica removeReplica(int brokerId, TopicPartition tp) {
     return leadershipLoad;
   }
 
+  /**
+   * (1) Make the replica with the given topicPartition and brokerId the leader.
+   * (2) Add the outbound network load associated with leadership to the given replica.
+   * (3) Add the CPU load associated with leadership.
+   *
+   * @param brokerId Id of the broker containing the replica.
+   * @param topicPartition TopicPartition of the replica for which the outbound network load will be added.
+   * @param resourceToLeadershipLoadBySnapshotTime Resource to leadership load to be added by snapshot time.
+   */
   void makeLeader(int brokerId,
-                  TopicPartition tp,
-                  Map<Resource, Map<Long, Double>> leadershipLoadBySnapshotTime) throws ModelInputException {
+                  TopicPartition topicPartition,
```

Comment:
Usually we want to avoid making the variable name too long if the meaning is clear enough. This will make the code look short and easier to read.

Reply:
Fixed.